user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Other/Timeline_component.md

```diff
@@ -0,0 +1,130 @@
+
+1. Optionally, hover the mouse pointer over the component, click the filter icon, and then add a filter feed from the *Feeds > URL > Query columns* section of the data pane. You can repeat this several times in order to filter on several query columns.
+
+1. To add item selection and group selection to the timeline component simultaneously<!--35638-->:
+
+   1. Hover the mouse pointer over the component and click the ![Groups](~/user-guide/images/NewRD_Groups.png) icon.
+
+      In the data pane on the right, any data feeds that cannot be added will become unavailable. Data feeds that are compatible will be marked with the following icon: ![available groups](~/user-guide/images/Group_Icon.png)
+
+   1. Drag the compatible data feed onto the component.
+
+      > [!NOTE]
+      > If you add a query column to the timeline component that contains data where the start and end time are empty, these empty groups are still displayed<!--RN 35600-->.
+
+   1. Add a feed from the *Feeds > Timeline # > Selected groups > Query rows* section of the data pane.
+
+1. Optionally, customize the following component options:
+
+   - *WebSocket settings*: Determines whether the websocket settings configured in the page/panel settings should be applied to this component. Enabled by default.
+
+   - *General > Timeline*: Allows you to configure the start and end times of the timeline component. You can choose between the following options: *Start*, *End*, and *Discount*.
+
+   - *General > Override dynamic units*: Disables parameter units from changing dynamically based on their value and protocol definition. Disabled by default.
+
+   - *General > Use dynamic units*: When the *Override dynamic units* option is enabled, this option will allow you to determine whether parameter units will change dynamically based on their value and protocol definition.
+
+   - *General > Default time range*: Allows you to select an option with a particular time to zoom to this time on the timeline, e.g. Today, Last 7 days, Next hour, etc. The options are divided in the following categories: *Still busy*, *In the past*, *Near future*, *Recently*, *Long run*, *Starting from now*, and *Distant future*. Set to *Still busy, This week* by default<!--RN 33287-->.
+
+     If you select *Custom*, you can set a custom start and end time.
+
+     ![Custom time range](~/user-guide/images/Default_Time_Range.png)<br/>*Settings timeline component in DataMiner 10.4.1*
+
+     > [!NOTE]
+     > The component has a minimum time range of 5 milliseconds and a maximum of 10 years<!--RN 35620-->.
+
+     To use the default time range configured for another component in the dashboard or low-code app, click the ![Link to feed](~/user-guide/images/Link_to_Feed.png) icon next to *Link time range to feed* and select the component from the dropdown list. Modifying the default time frame for this component will automatically synchronize the time range for any linked components as well.
```

Comment:
The "link to feed" has nothing to do with the default time range. When you link the timeline to eg a time range component, the timeline will show the range that is selected by that linked time range component. When you would for example switch the timerange in the timerange component to "this week", the timeline will immediatly also switch to this week.